content/en/docs/tasks/security/mtls-migration/index.md

```diff
@@ -75,6 +75,24 @@ down once the migration is done.
 
 ## Configure clients to send mutual TLS traffic
 
+Configure Istio services to send mutual TLS traffic by setting `Policy`, whose mode is `PERMISSIVE`.
```

Comment:
Are this needed? We ship with default installation including PERMISSIVE?

Can you describe your installation command line, along with the version you used?

Reply:
You are right, this is not necessary since ship with installation including PERMISSIVE.
However, it will make user confuse, since the document did not mention this.
So, as I said, I can cancel this pull request, had better mentioned this in document in another new pull request.

Reply:
Just FYI.
I will cancel this if you think the document is clear enough.